framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -525,10 +526,13 @@ private void applyDefaultsElement(final Element annotationScope, final Annotated
                 // but it doesn't work correctly and tests fail.
                 // (That whole @FromStubFile and @FromByteCode annotation
                 // logic should be replaced by something sensible.)
-            if ((ElementUtils.isElementFromByteCode(annotationScope) &&
+        	SourceChecker checker = atypeFactory.getContext().getChecker();
+        	if ((ElementUtils.isElementFromByteCode(annotationScope) &&
                     atypeFactory.declarationFromElement(annotationScope) == null &&
-                    !atypeFactory.isFromStubFile(annotationScope)) ||
-                    !annotatedForThisChecker
+                    !atypeFactory.isFromStubFile(annotationScope) &&
```

Comment:
These lines are all left-indented, making the structure of the conditional not obvious from looking at the code.
Please rewrite so that all elements of a given conjunction or disjunction are at the same indentation level.